test/unit/signup/account_manager_test.rb

```diff
@@ -258,6 +267,15 @@ class DeveloperAccountManagerTest < Signup::AccountManagerTest
         assert_equal 'created', account.state
       end
 
+      test 'create developer without email' do
+        signup_result = signup_account_manager.create(signup_params(different_account_params: { email: '' }))
+
+        refute signup_result.valid?
+        refute signup_result.persisted?
+        signup_errors = signup_result.errors.full_messages
+        assert_includes signup_errors, 'Email can\'t be blank'
+      end
```

Comment:
I'm not sure if writing the same test twice is a good idea. Plus, the change has been done in the User model, so perhaps we should only have a test in that model (in `test/unit/user_test.rb` )

Reply:
Should it go outside of the class? I guess there should be one per "block", no? `ProviderAccountManagerTest` and `DeveloperAccountManagerTest`?